users/tests/test_user_signup.py

```diff
@@ -0,0 +1,40 @@
+from rest_framework.test import APITestCase, APIRequestFactory
+from rest_framework import status
+from users.models import User
+from users.tests.factories import UserFactory
+import factory, faker
+
+class SignUpTests(APITestCase):
+    def setUp(self):
+        self.data=factory.build(dict, FACTORY_CLASS=UserFactory)
+        self.password=faker.Faker().password()
+
+    def test_sign_up_success(self):
+        self.data.update({'password1': self.password, 'password2': self.password})
+        response = self.client.post('/dj-rest-auth/registration/', self.data)
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        self.assertEqual(User.objects.count(), 1)
+        self.assertEqual(User.objects.get().first_name, self.data['first_name'])
+        self.assertEqual(User.objects.get().last_name, self.data['last_name'])
+        self.assertEqual(User.objects.get().gender, self.data['gender'])
```

Comment:
I think it would be better to get the user once instead of accessing the db each time. What do you think?
```suggestion
        db_user = User.objects.get()

        self.assertEqual(User.objects.count(), 1)
        self.assertEqual(db_user.first_name, self.data['first_name'])
        self.assertEqual(db_user.last_name, self.data['last_name'])
        self.assertEqual(db_user.gender, self.data['gender'])
```